aepp/exportDatasetToCloudStorage.py

```diff
@@ -0,0 +1,179 @@
+import flowservice
+import destinationinstanceservice
+import os
+import time
+from utils import Utils
+
+class ExportDatasetToCloudStorage:
+    global flow_conn
+    global dis_conn
+    global utils
+    global username
+    def __init__(self):
+        self.username = os.getlogin()
+        self.flow_conn = flowservice.FlowService()
```

Comment:
I will offer to pass the config file to generate the instance of the FlowService class instead of implying the latest config load is the one you want to connect. 
That would help on cross sandbox and cross IMS setup. 

You can have a look on how instanciation of FlowService is done, see how the `config` parameter is passed. It can be either a dictionary or an instance of the ConnectInstance class.

Reply:
Hi @pitchmuc, thanks for the feedback. I think it is a good idea to add.

Reply:
Added both config and header option when calling __init__.